sechub-cli/src/daimler.com/sechub/cli/controller.go

```diff
@@ -22,21 +22,17 @@ type jobScheduleResult struct {
 // Execute starts sechub client
 func Execute() {
 
-	printLogoWithVersion(os.Stdout)
-
 	context := InitializeContext()
 
+	printLogoWithVersion(context)
+
 	switch context.config.action {
 	case scanAction:
-		{
```

Comment:
I believe something broke with the -help parameter.

If I build this branch and then run `sechub -help` I get an error message:
`ERROR: Given SecHub config file 'sechub.json' is not correct JSON!`

My earlier version of the client is printing the help. Please check and fix this.